tests/brevitas/graph/test_channel_splitting.py

```diff
@@ -0,0 +1,139 @@
+from pytest_cases import fixture_union
+import torch
+from torchvision import models
+
+from brevitas.fx import symbolic_trace
+from brevitas.graph.fixed_point import MergeBatchNorm
+from brevitas.ptq_algorithms.channel_splitting import *
+
+from .equalization_fixtures import *
+
+
+@pytest.mark.parametrize('split_ratio', [0.05, 0.1, 0.2])
+@pytest.mark.parametrize('split_input', [False, True])
+def test_resnet18(split_ratio, split_input):
```

Comment:
Check the equalization tests, and re-use some of the utilities there.

Reply:
Let's have 2 group of tests:

- One for the torchvision models
- One for the toy examples

Mark with `pytest.skip` everything that is not compatible with channel splitting. Modify the tests to merge batch norm only for channel splitting (just because otherwise we would need to update all the other tests).